packages/wallets/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -1,34 +1,37 @@
 import React, { useState } from 'react';
 import { useAvailableMT5Accounts } from '@deriv/api';
-import { ModalStepWrapper, WalletButton, WalletText } from '../../../../components/Base';
+import { ModalStepWrapper, WalletButton } from '../../../../components/Base';
 import { useModal } from '../../../../components/ModalProvider';
 import { JurisdictionScreen } from '../../screens/Jurisdiction';
 import { MT5PasswordModal } from '..';
 
+const marketTypeToTitleMapper = {
+    all: 'Swap-Free',
+    financial: 'Financial',
+    synthetic: 'Derived',
```

Comment:
idea for future: that sounds like translation responsibility, would it be possible to squeeze that to translations files instead of adding mapping logic within JavaScript?

Reply:
Yes. Will improve this in the upcoming PR's